ansible/roles/common/tasks/main.yml

```diff
@@ -28,15 +28,13 @@
   apt:
     update_cache: yes
 
-- name: install make, gcc, librados-dev and librbd-dev external packages
+- name: install make and gcc external packages
```

Comment:
Why are librados-dev and librbd-dev removed?  I thought they are required for CSI testing?

Reply:
They will installed when building opensds source code with `make` command.

Reply:
Ok